website_docs/getting_started.md

```diff
@@ -25,7 +25,7 @@ gem 'opentelemetry-exporter-otlp'
 gem 'opentelemetry-instrumentation-all'
 ```
 
-The inclusion of `opentelemetry-instrumentation-all` in the above list provides instrumentations for several frameworks such as Rails and Sinatra as well as database drivers and HTTP [libraries][auto-instrumentation].
+The inclusion of `opentelemetry-instrumentation-all` provides [instrumentations](automatic_instrumentation.md) for Rails, Sinatra, several HTTP libraries, and more.
```

Comment:
Please use anchors and relative paths without qualifying pages with `.md` extension. 

https://github.com/open-telemetry/opentelemetry-ruby/blob/main/website_docs/getting_started.md?plain=1#L86